### PR TITLE
Qol

### PR DESCRIPTION
**Pull request by gh-23378**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,44 +4,53 @@
...
+3. On the network dropdown at the top of the MetaMask popup, select show/hide test networks and toggle on test networks
+4. Change your network to localhost:8545
+5. Inside of settings > networks > localhost:8545, change the localhost:8545 network’s chain id to 31337
+6. Start a local hardhat node (see the v2 contracts repo for more detailed instructions)
+7. Import the private key for an account funded with SARCO and Ether on the local hardhat network.
+    - The first account generated when the node is run from the v2 contracts project will be funded. Its private key is `0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d`
+8. Ensure that at least one archaeologist has been registered on your local hardhat node
+    - If your node is restarted, you will need to reregister your archaeologist
+    - Your archaeologist node must be pointed at ArLocal which runs at 127.0.0.1:1984 by default
+
```
Comment (DarksightKellar) [inline README.md:54]: Whew this is thorough! 🙌🏾

**Review by DarksightKellar — APPROVED** [review]

**Review by DarksightKellar — CHANGES_REQUESTED** [review]
Looks good! There's linter complaints though causing netlify build to fail

File: README.md (hunk 1 above)
Comment (sethhrbek) [reply README.md:54]: The arch will automatically be pointed at arLocal when running on the local network (doesn't hurt to still have this note, but user doesn't need to take any action for this)

**Review by sethhrbek — APPROVED** [review]

**Comment by sethhrbek** [comment]
these are the linter errors --

image

**Review by DarksightKellar — APPROVED** [review]

**Pull request merged**